cookbook-efforts/domain-specific-datasets/README.md

```diff
@@ -20,22 +20,21 @@ The goal of this project is to share and collaborate with domain experts to crea
 
 🧑🏻‍🔧 If you are an (inspiring) Machine Learning engineer, you can set up the project and its tools. You can run the synthetic data generation pipelines. And maybe even get around to training models.
 
-
```

Comment:
I think I might be missing this uniform introduction accross these cookbooks? What/why/how etc? I believe this only shows the how.